app/src/main/java/de/danoeh/antennapod/adapter/StatisticsListAdapter.java

```diff
@@ -0,0 +1,105 @@
+package de.danoeh.antennapod.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.joanzapata.iconify.widget.IconTextView;
+
+import java.util.ArrayList;
+import java.util.Locale;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.feed.Feed;
+import de.danoeh.antennapod.core.glide.ApGlideSettings;
+import de.danoeh.antennapod.core.storage.DBReader;
+
+/**
+ * Adapter for the statistics list
+ */
+public class StatisticsListAdapter extends BaseAdapter {
+    private Context context;
+    ArrayList<DBReader.StatisticsItem> feedTime = new ArrayList<>();
+
+    public StatisticsListAdapter(Context context) {
+        this.context = context;
+    }
+
+
+    @Override
+    public int getCount() {
+        return feedTime.size();
+    }
+
+    @Override
+    public DBReader.StatisticsItem getItem(int position) {
+        return feedTime.get(position);
+    }
+
+    @Override
+    public long getItemId(int position) {
+        return 0;
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+        FeedHolder holder;
+        Feed feed = feedTime.get(position).feed;
+
+        if (convertView == null) {
+            holder = new FeedHolder();
+            LayoutInflater inflater = (LayoutInflater) context
+                    .getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+
+            convertView = inflater.inflate(R.layout.nav_feedlistitem, parent, false);
```

Comment:
While this creative re-use of a layout, I'm not sure this is a good idea. You create an additional, non-obvious dependency here. Whenever we change the navbar listitem layout in the future, we would also have to consider that the layout is also used here.
You are also not using failure at all and using the former count (of new/unused/unused+new episodes) for the total runtime also has a rather bad smell to it. You should at least give your variables better names!